### PR TITLE
Issue: Extra spacing between top of the page and the header on variou…

### PR DESCRIPTION
**Pull request by notandrew**
…s Components pages

Fix: Replacing \r\b with \r\n in ComponentSamples.js causes space between top of the page and the header to go away.
+ Icons page is mostly updating the CSS to be consistent with Components pages, spacing changes, and code style updates

### CODE REVIEW

File: src/samples/Icons/index.html [hunk 1]
```diff
@@ -59,363 +68,371 @@
...
-                    <p class="ms-font-l"><i class="ms-Icon ms-Icon--mail"></i></p>
-                    <p class="ms-font-l">Note the <code>aria-hidden</code> attribute, which prevents screen readers from reading the icon. In cases where meaning is conveyed <em>only</em> through the icon, such as an icon-only navigation bar, be sure to apply an <code>aria-label</code> to the button for accessibility.</p>
-                </aside>
-            </div>
+<body>
+    <section class="docs-Section">
+        <div class="docs-Header">
+            <h1 class="ms-font-su ms-fontColor-white ms-bgColor-themeDarker">Icons</h1>
+            
+            <p class="docs-Intro ms-font-l">Fabric uses a custom font for its iconography, containing glyphs that can be infinitely scaled, colored, styled, and even flipped for right-to-left localization.</p>
```
Comment (wdo3650) [inline src/samples/Icons/index.html]: Not sure about this extra space. It sort of looks off on the page.

File: src/samples/Icons/index.html [hunk 2]
```diff
@@ -1,57 +1,66 @@
...
+            margin: 0;
+            padding: 10px;
             margin-bottom: 20px;
+            padding-left: 25px;
+        }
+        
+        .docs-Intro,
+        .docs-Example, 
+        .docs-IconsSection {
+            margin: 3vw;
```
Comment (wdo3650) [inline src/samples/Icons/index.html]: Not sure about the top and bottom margin for this. The spacing between the intro and the content below seems off.

**Comment by wdo3650** [comment]
I like what you got! Good job. Approved.

**Comment by mikewheaton** [comment]
Looks good! Let's merge this in.

**Pull request merged**